src/utils/optvariable.jl

```diff
@@ -39,7 +40,8 @@ function OptVariable(cep::OptModelCEP,
       end
     end
   end
-  OptVariable(round.(jumparray.data;digits=round_sigdigits),jumparray.axes...; axes_names=axes_names, type=type)
+  unscaled_data=jumparray.data*scale[variable] #Unscale the jumparray data based on the scaling parameters in Dictionary scale
```

Comment:
I am not entirely sure if I follow fully. You unscale the optimal variables here and store them. Where do you scale them previously?

Reply:
And does that mean that if I just take the optimization results as is form the JuMP model, the variable units would not be correct, right?

Reply:
The JuMP variables and equations are scaled within the problem formulation (one can not scale them separately like in GAMS - idea being to give you more control of what is actually given to optimizer)

The JuMP variables themselves are scaled and unscaling happens afterwards.

Reply:
Ok, I see. Let's walk through this in person together, I think this is a great change in terms of computational time reduction, but needs to be documented well, because the units do not hold anymore. Make sure to document this in the function header and also where scale is applied: `What is the reason for using scale, where is it unscaled`.